homeassistant/components/laundrify/__init__.py

```diff
@@ -0,0 +1,50 @@
+"""The laundrify integration."""
+from __future__ import annotations
+
+import logging
+
+from laundrify_aio import LaundrifyAPI
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[str] = ["binary_sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up laundrify from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+    hass.data[DOMAIN][entry.entry_id] = {}
+
+    # Registers update listener to update config entry when options are updated.
+    unsub_options_update_listener = entry.add_update_listener(options_update_listener)
```

Comment:
I think you can even simplify this more by using this:
```
    entry.async_on_unload(entry.add_update_listener(options_update_listener))
```.
Than you don't need to save and call it again.

Reply:
Thanks. Fixed in 042e2af582b599b54570553a7003091042de86cf